homeassistant/components/bryant_evolution/climate.py

```diff
@@ -38,12 +35,20 @@ async def async_setup_entry(
     # Add a climate entity for each system/zone.
     sam_uid = names.sam_device_uid(config_entry)
     entities: list[Entity] = []
+    coordinator = EvolutionCoordinator(
+        hass, config_entry.data[CONF_FILENAME], config_entry.data[CONF_SYSTEM_ZONE]
+    )
+
+    # Fetch initial data so we have data when entities subscribe.
+    await coordinator.async_config_entry_first_refresh()
+
```

Comment:
As we talked already, the coordinator should preferably be instantiated in `__init__.py` and stored in runtime_data

Reply:
done